testgrid/conformance/upload_e2e.py

```diff
@@ -32,6 +32,7 @@
 #
 # Usage: see README.md
 
+from __future__ import print_function
```

Comment:
> Note: this is required for pylint: 1.9.4 to understand python3 print function.

Reply:
Can you drop this into the file as a comment?

Reply:
Done. 👍